Helm/opensearch-dashboards/templates/role.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "opensearch-dashboards.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
```

Comment:
Other templates put this on one line and use `nindent`. Shoud it be the same here?

In general there's quite an inconsistent mix of `indent`/`nindent` usage in the templates.

Reply:
I feel the usage of `indent`/`nindent` are very much subjective. Is there anything which you think is majorly disturbing and you want me to change?